waves/src/Bobtimus.tsx

```diff
@@ -56,14 +51,14 @@ export interface LoanOffer {
     max_ltv: number;
     base_interest_rate: number;
     terms: Term[];
-    collateralizations: Collateralization[];
+    initial_ltvs: Ltv[];
 }
 
 export interface LoanRequest {
     principal_amount: number;
     collateral_amount: number;
     collateral_inputs: { txin: OutPoint; original_txout: any; blinding_key: string }[];
-    collateralization: number;
+    ltv: number;
```

Comment:
Why does the borrower send an LTV? 
The current LTV of the borrower can be calculated from `=REPAYMENT_AMOUNT/(LOAN_REQUEST_COLLTERAL*CURRENT_RATE)`. This feels weird, because the borrower now sends values that are used to calculate the LTV, but at the same time provides an LTV. Is this just to cross check that the calculation on UI and Bobtimus is the same?